src/index.html

```diff
@@ -1,12 +1,434 @@
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Company logo"
+          class="logo__image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <catalog class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__product-img"
+            src="./images/imac.jpeg"
+            alt="iMac"
+          >
+
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <p class="stars__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__product-price">
+            <p class="card__product-price_text">
+              Price:
+            </p>
+            <p class="card__product-price_value">
+              &dollar;2,199
+            </p>
+          </div>
+
+          <a
+            class="card__purchase-link"
+            data-qa="card-hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+        <div class="card">
```

Comment:
Add a blank line between the cards
```suggestion
        </div>
        
        <div class="card">
```